cypress/integration/examples/cookies.spec.js

```diff
@@ -0,0 +1,77 @@
+/// <reference types="cypress" />
+
+context('Cookies', () => {
+  beforeEach(() => {
+    Cypress.Cookies.debug(true)
```

Comment:
The test file is not testing the Book Store app as stated in the task description. Consider updating the test cases according to the given task.